_includes/layouts/post.njk

```diff
@@ -0,0 +1,10 @@
+---
+layout: layouts/base.njk
+templateClass: tmpl-post
+---
+<h1>{{ title }}</h1>
+
+{% if author %}
+By <a href="{{ authorLink }}">{{ author }}</a>
+{% endif %}
+{{ content | safe }}
```

Comment:
Not necessary for this PR _but_ it would be nice if we could have a readable dates on the actual posts as well.